controller/watcher.go

```diff
@@ -124,3 +130,38 @@ func setVaultClient(clients *hcat.ClientSet, conf *config.Config) error {
 
 	return clients.AddVault(vault)
 }
+
+func newWatcherEventHandler() events.EventHandler {
+	logger := logging.Global().Named(hcatLogSystemName)
+	return func(e events.Event) {
+		// Display events at different log levels.
```

Comment:
Idea to make this comment a bit clearer:
// Log events at different log levels based on the event